src/graphinfo.jl

```diff
@@ -63,7 +63,8 @@ function Model(;kwargs...)
     for (i, node) in enumerate(values(kwargs))
         @assert typeof(node) <: Tuple{Union{Array{Float64}, Float64}, Function, Symbol} "Check input order for node $(i) matches Tuple(value, function, kind)"
     end
-    vals = getvals(NamedTuple(kwargs))
+    vals = [getvals(NamedTuple(kwargs))...]
+    vals[1] = Tuple([Ref(Tuple(val)) for val in vals[1]])
```

Comment:
Why we are wraping `val` inside a `Tuple`? 
```suggestion
    vals[1] = Tuple([Ref(val) for val in vals[1]])
```

Reply:
I think otherwise `val` is a vector. Are we happy to keep `val` as a vector? I have no objections.

Reply:
Probably easier to work with vectors for vals...